RELEASE-NOTES.md

```diff
@@ -7,9 +7,9 @@ See the documentation (http://doc.splicemachine.com) for the setup and use of th
 
 Supported platforms with 2.5:
 
-* Cloudera CDH 5.7.2, 5.8.0, 5.8.3
-* MapR 5.1.0 and 5.2.0
-* HortonWorks HDP 2.4.2 and 2.5
+* Cloudera 5.8.3, 5.12.0, 5.12.2, 5.13.2, 5.14.0
+* MapR 5.2.0
+* HortonWorks HDP 2.6.3
```

Comment:
why did HDP 2.5.5 fall off this list? we've still got a paying customer using this version...